[2023-11-30T03:15:16.528+0200] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: lviv_weather.inject_data_Zhmerynka manual__2023-11-30T01:15:03.402311+00:00 [queued]>
[2023-11-30T03:15:16.541+0200] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: lviv_weather.inject_data_Zhmerynka manual__2023-11-30T01:15:03.402311+00:00 [queued]>
[2023-11-30T03:15:16.542+0200] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-30T03:15:16.557+0200] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): inject_data_Zhmerynka> on 2023-11-30 01:15:03.402311+00:00
[2023-11-30T03:15:16.562+0200] {standard_task_runner.py:57} INFO - Started process 171596 to run task
[2023-11-30T03:15:16.567+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'lviv_weather', 'inject_data_Zhmerynka', 'manual__2023-11-30T01:15:03.402311+00:00', '--job-id', '234', '--raw', '--subdir', 'DAGS_FOLDER/lviv_weather.py', '--cfg-path', '/tmp/tmp3kns7it7']
[2023-11-30T03:15:16.569+0200] {standard_task_runner.py:85} INFO - Job 234: Subtask inject_data_Zhmerynka
[2023-11-30T03:15:16.668+0200] {task_command.py:416} INFO - Running <TaskInstance: lviv_weather.inject_data_Zhmerynka manual__2023-11-30T01:15:03.402311+00:00 [running]> on host yurabd.
[2023-11-30T03:15:17.151+0200] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='lviv_weather' AIRFLOW_CTX_TASK_ID='inject_data_Zhmerynka' AIRFLOW_CTX_EXECUTION_DATE='2023-11-30T01:15:03.402311+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-30T01:15:03.402311+00:00'
[2023-11-30T03:15:17.152+0200] {sql.py:274} INFO - Executing: 
            INSERT INTO measures (city, timestamp, date, temp, humidity, clouds, wind_speed) VALUES
            ('Zhmerynka',
            to_timestamp(1701306903),
            '2023-11-30 01:15:03',
            -5.27,
            86,
            7,
            3.74);
            
[2023-11-30T03:15:17.167+0200] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-11-30T03:15:17.220+0200] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-11-30T03:15:17.344+0200] {sql.py:418} INFO - Running statement: 
            INSERT INTO measures (city, timestamp, date, temp, humidity, clouds, wind_speed) VALUES
            ('Zhmerynka',
            to_timestamp(1701306903),
            '2023-11-30 01:15:03',
            -5.27,
            86,
            7,
            3.74);
            , parameters: None
[2023-11-30T03:15:17.345+0200] {sql.py:427} INFO - Rows affected: 1
[2023-11-30T03:15:17.360+0200] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=lviv_weather, task_id=inject_data_Zhmerynka, execution_date=20231130T011503, start_date=20231130T011516, end_date=20231130T011517
[2023-11-30T03:15:17.386+0200] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-30T03:15:17.411+0200] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
